gulpconfig.js

```diff
@@ -16,6 +16,17 @@ var options = {
         // Include paths to resolve automatically.
         includePaths: ['./bower_components/']
       }
+    },
+    {
```

Comment:
No need to create a separate config section for this.  Any non-partials in the scss directory will be built automatically.